consonance-webservice/src/main/java/io/swagger/wes/api/impl/Ga4ghApiServiceImpl.java

```diff
@@ -0,0 +1,414 @@
+package io.swagger.wes.api.impl;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.google.common.collect.Lists;
+import com.google.gson.Gson;
+
+import io.consonance.arch.Base;
+import io.consonance.arch.beans.JobState;
+import io.consonance.arch.persistence.PostgreSQL;
+import io.consonance.client.WebClient;
+import io.consonance.common.CommonTestUtilities;
+import io.consonance.common.Constants;
+import io.consonance.common.Utilities;
+import io.dockstore.client.cli.Client;
+import io.dockstore.client.cli.nested.AbstractEntryClient;
+import io.dropwizard.hibernate.UnitOfWork;
+import io.swagger.client.ApiException;
+import io.swagger.client.api.OrderApi;
+
+import io.swagger.client.model.ExtraFile;
+import io.swagger.client.model.SourceFile;
+import io.swagger.models.auth.In;
+import io.swagger.wes.api.*;
+import io.swagger.wes.model.*;
+
+import io.consonance.arch.beans.Job;
+
+import io.swagger.wes.model.Ga4ghWesServiceInfo;
+import io.swagger.wes.model.Ga4ghWesWorkflowListResponse;
+import io.swagger.wes.model.Ga4ghWesWorkflowLog;
+import io.swagger.wes.model.Ga4ghWesWorkflowRequest;
+import io.swagger.wes.model.Ga4ghWesWorkflowRunId;
+import io.swagger.wes.model.Ga4ghWesWorkflowStatus;
+
+import java.io.*;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
```

Comment:
FYI, on behalf of future-you. I'd configure your IDE to avoid wildcard imports. 
Explanation here https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad